examples/web/index.html

```diff
@@ -1,6 +1,8 @@
 <!DOCTYPE html>
 <html>
 <head>
+  <base href="/flame/">
```

Comment:
shouldn't we deploy this under flame-engine.org/examples ?

Reply:
I was thinking on doing this on a next step, I wanted to make sure that the config worked first, then on a follow up, do that, specially because I will need your help on doing so

Reply:
not sure I understand this line then. but no big deal either way

Reply:
This is a config required by Flutter when you are not deploying the site directly on the "root" of the domain. When we change for the url you suggested, we will need to change it to `examples`

Reply:
what I didn't understand is in which URL this will be deployed that has a `/flame` subpath

Reply:
https://flame-engine.github.io/flame/

Reply:
ahh gotcha